keylime/ca_util.py

```diff
@@ -364,7 +364,7 @@ def read_private(warn: bool = False) -> Tuple[Dict[str, Any], str]:
         key = crypto.kdf(global_password, toread["salt"])
         try:
             plain = crypto.decrypt(toread["priv"], key)
-        except ValueError as e:
+        except crypto_exceptions.InvalidTag as e:
```

Comment:
that is a good question ... but because of the nature of Python we do not know the set of those exception in the multiple execution path that `.decrypt()` have

IMHO this is good for now, as the exception not catch here will be throw in any case, and later exposed

Reply:
+1 on this. It is an incremental improvement.

@kaifeng Good to see you back here.